apps/testnorge-arena/arena-core/src/main/java/no/nav/registre/arena/core/service/VedtakshistorikkService.java

```diff
@@ -146,6 +151,14 @@
         return rettighetArenaForvalterConsumer.opprettRettighet(serviceUtils.opprettArbeidssoekerAap(rettigheter, miljoe));
     }
 
+    private List<NyttVedtakTillegg> fjernTilsynFamiliemedlemmerVedtakMedUgyldigeDatoer(List<NyttVedtakTillegg> tilsynFamiliemedlemmer) {
+        var nyTilsynFamiliemedlemmer = tilsynFamiliemedlemmer.stream().filter(vedtak ->
```

Comment:
Jeg tror du må ha en nulltest på tilsynFamiliemedlemmer-lista for å unngå NPE her

Reply:
Godt sett, antagelig derfor jeg fikk NPE når test skulle kjøres!